rpcs3/Emu/Io/GHLtar.h

```diff
@@ -0,0 +1,15 @@
+﻿#pragma once
+
+#include "Emu/Io/usb_device.h"
+#include <queue>
+#include <mutex>
```

Comment:
are queue and mutex really needed?

Reply:
Just kinda copied from Skylander.h and forgot